src/test/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointAlleleTest.java

```diff
@@ -32,4 +32,23 @@ public void testIsInversion() throws Exception {
 
     }
 
+
+    @Test
+    public void testEqualsAndHashCode() throws Exception {
+
+        final AlignmentRegion region1 = new AlignmentRegion("1", "contig-1", TextCigarCodec.decode("100M"), true, new SimpleInterval("1", 10000, 10100), 60, 1, 100, 0);
+        final AlignmentRegion region2 = new AlignmentRegion("1", "contig-1", TextCigarCodec.decode("100M"), false, new SimpleInterval("1", 20100, 20200), 60, 101, 200, 0);
+        final ArrayList<String> insertionMappings1 = new ArrayList<>();
+        insertionMappings1.add("foo");
+        final BreakpointAlignment breakpoint1 = new BreakpointAlignment("contig-1", region1, region2, "", "ACAC", insertionMappings1);
+
+        final AlignmentRegion region3 = new AlignmentRegion("2", "contig-2", TextCigarCodec.decode("100M"), true, new SimpleInterval("1", 10000, 10100), 60, 1, 100, 0);
+        final AlignmentRegion region4 = new AlignmentRegion("2", "contig-2", TextCigarCodec.decode("100M"), false, new SimpleInterval("1", 20100, 20200), 60, 101, 200, 0);
+        final ArrayList<String> insertionMappings2 = new ArrayList<>();
+        insertionMappings2.add("bar");
+        final BreakpointAlignment breakpoint2 = new BreakpointAlignment("contig-2", region3, region4, "", "ACAC", insertionMappings2);
+
+        Assert.assertEquals(breakpoint1.getBreakpointAllele(), breakpoint2.getBreakpointAllele());
+        Assert.assertEquals(breakpoint1.getBreakpointAllele().hashCode(), breakpoint2.getBreakpointAllele().hashCode());
+    }
```

Comment:
You could consider adding another test that exercised serialization (though I won't insist on it).  See `BreakpointEvidenceTest.serializationTest()` for inspiration.